src/vs/workbench/parts/debug/browser/debugStatus.ts

```diff
@@ -47,7 +50,13 @@ export class DebugStatus extends Themable implements IStatusbarItem {
 				this.showInStatusBar = configurationService.getValue<IDebugConfiguration>('debug').showInStatusBar;
 				if (this.showInStatusBar === 'never' && this.statusBarItem) {
 					this.statusBarItem.hidden = true;
-				} else {
+				}
```

Comment:
can you put the `else` branch on the same line with the closing bracket. I just personally prefer it that way